parm/config/config.wave

```diff
@@ -69,8 +75,21 @@ export FHINCP_WAV=`expr $DTPNT_WAV / 3600`
 export OUTPARS_WAV="WND CUR ICE HS T01 T02 DIR FP DP PHS PTP PDIR CHAR"
 # GFS # export OUTPARS_WAV='WND CUR ICE HS T01 T02 DIR FP DP PHS PTP PDIR CHAR'
 
+# Subgrid info for grib2 encoding
+export WAV_SUBGRBSRC="gnh_10m"
+export WAV_SUBGRB="WAV_ATLO_GRB WAV_EPAC_GRB WAV_WCST_GRB"
+export WAV_ATLO_GRB="0 6 0 0 0 0 0 0 301 331 0 0 55000000 260000000 48 0 310000000 166667 166667 0 atlocn 0p16"
+export WAV_EPAC_GRB="0 6 0 0 0 0 0 0 511 301 0 0 30000002 130000000 48 -20000000 215000000 166667 166667 0 epacif 0p16"
+export WAV_WCST_GRB="0 6 0 0 0 0 0 0 241 151 0 0 50000000 210000000 48 25000000 250000000 166667 166667 0 wcoast 0p16"
+
 # Options for point output (switch on/off boundary point output)
-export DOIBP_WAV='NO'
+export DOIBP_WAV='NO' # Input boundary points
+export DOFLD_WAV='YES' # Field data
```

Comment:
While we may need these DO*WAV variables to be YES for the official GFS v16 parallels, do we want these variables to be YES in the repository?   Developers will check out this branch and run parallels.   Many may not need this output.   We set some downstream variables to NO (e.g., DO_GEMPAK and DO_AWIPS) for this very reason.

Reply:
My understanding is that config.wave is only used at the forecast step.  The following DO*_WAV are for post-processing and production generation.  Shouldn't hey be moved to config.wavepostsbs ?

Reply:
config.wave need to be used by all wave jobs/scripts as it contains general parameters that need to be consistently used across the wave component workflow steps. That said. yes some need to be moved to step-specific config files. I would prefer to complete this in the T2O to ensure enough time to make the changes carefully and thoroughly. I will nevertheless move as much as I can. Changes will appear in a forthcoming push.

Reply:
Done

Reply:
Henrique moved some variables out of config.wave. Remaining variables in config.wave cover all wave jobs and the wave coupled fcst. Will leave further extraction and possible creation of config.wavefcst to T20. Doesn't seem necessary at this point...yet.